_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP and why is it so important to do a pre-GSoC analysis
+and design it as early as possible, for it to be finalized in the cEP?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the **coala** community
+to discuss the new *major features* that the contributor intends to implement
+during the GSoC duration. The proposal is designed before the implementation of
+*major features* because the implementation usually happens from a technical
+perspective which may lead to a poorly designed content - developer and user-wise
+, so to avoid this, a cEP is designed at an earlier stage that can essentially
```

Comment:
`so to avoid this` should start a new sentence. Its always hard to keep longer sentences grammatically correct and coherent :slightly_smiling_face:

Reply:
Yeah when i was forming that sentence I was getting confused that whether to start a new sentence or just continue with that previous sentence. Then some thought came in my mind and I just continued with joining this sentence with previous one. Then I thought reviewers will help me out and you just helped me out. Thnx 😄